test/typed-function-references.wast.fromBinary.noDebugInfo

```diff
@@ -1,9 +1,15 @@
 (module
- (type $none_=>_none (func))
  (type $i32_=>_i32 (func (param i32) (result i32)))
+ (type $none_=>_none (func))
+ (type $none_=>_eqref (func (result eqref)))
+ (type $i32_i32_i32_i32_i32_=>_none (func (param i32 i32 i32 i32 i32)))
  (type $none_=>_i32 (func (result i32)))
+ (type $none_=>_anyref (func (result anyref)))
  (type $ref_null<i32_->_i32>_=>_i32 (func (param (ref null (func (param i32) (result i32)))) (result i32)))
  (type $ref<i32_->_i32>_=>_i32 (func (param (ref (func (param i32) (result i32)))) (result i32)))
+ (type $f64_=>_ref_null<_->_eqref> (func (param f64) (result (ref null (func (result eqref))))))
+ (type $none_=>_ref<i32i32i32i32i32_->_> (func (result (ref (func (param i32 i32 i32 i32 i32))))))
```

Comment:
It looks like this type name generation could be improved (but not necessarily in this PR).

Reply:
I think I found a reasonable way to improve this, pushed now. It emits things like `$none_=>_ref[i32_i32_->_]` (proper separation between elements, and using `[ ]` it is less confusing with the `>` in the `->`).